### PR TITLE
Enhancement/9600 cta banner spacing

### PR DESCRIPTION
**Pull request by ankitrox**
## Summary

Addresses issue:

- #9600 

## PR Author Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 5.2 and PHP 7.4.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.
- [ ] Ensure there are no unexpected significant changes to file sizes.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

**Review by aaemnnosttv — CHANGES_REQUESTED** [review]
Thanks @ankitrox – it's great to see some consistency come to these 😄 Aside from the few comments, I noticed one of the cases in AdBlocking Recovery looks off still. Would you have a look?
image

File: assets/sass/components/ads/_googlesitekit-ads-setup-banner.scss [hunk 1]
```diff
@@ -57,7 +57,7 @@
 			gap: 4px;
 			margin-bottom: 27px;
 
-			@media (min-width: $width-desktop + 1 + px) {
+			@media (min-width: $width-tablet + 1 + px) {
```
[the remark this answers is not in the document]
Comment (ankitrox) [reply assets/sass/components/ads/_googlesitekit-ads-setup-banner.scss]: Thanks @aaemnnosttv. 

I have updated all instances in PR to use the `$bp-nonMobile` variable.

File: assets/sass/components/adsense/_googlesitekit-adsense-ad-blocking-recovery.scss [hunk 2]
```diff
@@ -208,7 +208,11 @@
 			align-items: center;
 			display: flex;
 			flex-wrap: wrap;
-			gap: $grid-gap-desktop + 2px; // The reference designs implement a spacing of 26px between the actions.
+			gap: 4px;
+
+			@media (min-width: $width-tablet + 1 + px) {
+				gap: 10px;
```
[the remark this answers is not in the document]
Comment (ankitrox) [reply assets/sass/components/adsense/_googlesitekit-adsense-ad-blocking-recovery.scss]: That's a nice suggestion. I have [added two variables](https://github.com/google/site-kit-wp/pull/9849/files#diff-71fa429f5fb0c3b90b41166e67b2847b0c1ac178a4a19d3d61a2dbd66e684abdR359-R360) `$cta-gap-phone` and `$cta-gap-tablet` to be used for CTA flex gap.

**Comment by ankitrox** [comment]
> Thanks @ankitrox – it's great to see some consistency come to these 😄 Aside from the few comments, I noticed one of the cases in AdBlocking Recovery looks off still. Would you have a look? image

Thank you for reviewing the PR @aaemnnosttv.

For AdBlocking Recovery, I've added the gap of 30px and for tablet and above, it can be 10px as for the other setup CTA banners.

I have made this change and tested on mobile screen which looks fine to me (like the previous one).

Can you please let me know if these changes looks fine or if there is any further suggestion.

Thank you.

**Review by aaemnnosttv — APPROVED** [review]
Great, thanks @ankitrox !

**Pull request merged**